ferdinand-porsche-fern-fachhochschule-german.csl

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-AT" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Ferdinand Porsche Fern-Fachhochschule (German)</title>
+    <title-short>Ferdinand Porsche FernFH (German)</title-short>
+    <id>http://www.zotero.org/styles/ferdinand-porsche-fern-fachhochschule-german</id>
+    <link href="http://www.zotero.org/styles/ferdinand-porsche-fern-fachhochschule-german" rel="self"/>
+    <link href="http://libweb.anglia.ac.uk/referencing/harvard.htm" rel="documentation"/>
+    <author>
+      <name>Christian Focke</name>
+      <email>christian.focke@mail.fernfh.ac.at</email>
+    </author>
+    <contributor>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Based on the Harvard author-date style</summary>
+    <updated>2015-12-28T09:39:29+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de-AT">
+    <terms>
+      <term name="accessed" form="long">
+        <single>Abruf am</single>
+        <multiple>Abrufe am</multiple>
+      </term>
+      <term name="anonymous" form="short">
+        <single>o. V.</single>
+        <multiple>o. V.</multiple>
+      </term>
+      <term name="et-al" form="long">
+        <single>et al.</single>
+        <multiple>et al.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="symbol" initialize-with=". " delimiter=", "/>
+      <label form="short" prefix=", " text-case="lowercase"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" strip-periods="false"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter="/" delimiter-precedes-last="never" et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize="false" initialize-with="." name-as-sort-order="all"/>
```

Comment:
You have somewhat odd et-al settings: For one, setting an `et-al-subsequent-min` for the bibliography makes no sense (since subsequent doesn't exist there). Then, you do set et-al on both citation and bibliography below, but override them here and in line 61. 
Please set them just one and do so in cs:citation and cs:bibliography (i.e. l. 142 and l. 156 respectively). Let me know if this is unclear.